ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -134,6 +134,12 @@ export const WorkspaceAbout = fp.flow(withUserProfile(), withCdrVersions())
     return workspace;
   }
 
+  // update the component state AND the store with the new workspace object
+  updateWorkspaceState(workspace: WorkspaceData): void {
```

Comment:
not entirely sure this is the right logic.  What I'd really like is for the state to update automatically when `currentWorkspaceStore` updates somehow

Reply:
Optional: I think a more natural way to do this is probably to use the `withCurrentWorkspace()` HOC. Reference workspace via props. Updating the workspace store should trigger the entire component to rerender.

This is a pretty old class component though, so I expect that refactor may uncover some issues.